### PR TITLE
Recent Blogs section initial setup

### PR DESCRIPTION
**Pull request by Mohitraut07**
Recent Blogs Section created.

### CODE REVIEW

File: frontend/src/components/Card/index.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Button from '../Button';
+
+const Card = () => {
+  return (
+    <>
+    <div className="h-[500px]  w-[400px] my-1 mx-1rounded-lg bg-[#E6E6E6] hover:shadow-md hover:shadow-[gray] hover:scale-105 ">
```
Comment (AniruddhaGawali) [inline frontend/src/components/Card/index.jsx:7]: Dont fix the width in px

File: frontend/src/components/Button/index.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const Button = () => {
+  return (
+
+        <>
+        <button className='right-full px-4 my-2 mx-2 bg-gray-200 rounded-lg hover:bg-gray-400 w-32 h-10'>Read More..</button>
```
Comment (AniruddhaGawali) [inline frontend/src/components/Button/index.jsx:7]: use Props for text inside button not hard core it by "Read More..."

File: frontend/src/components/Card/index.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Button from '../Button';
+
+const Card = () => {
```
Comment (AniruddhaGawali) [inline frontend/src/components/Card/index.jsx:4]: the information inside the card like desc, title, read more link,img etc add it by props don't hardcore it

File: frontend/src/pages/index.js [hunk 4]
```diff
@@ -10,9 +12,19 @@ export default function Home() {
...
+      <Divider/>
+      <div>
+      <div className="text-4xl left-24 m-10 truncate align-middle text-gray-300 mb-5"> RECENT BLOGS </div>
+      <div className="flex justify-evenly items-center mb-10 mt-10">
+          <Card />
+          <Card />
+          <Card />
+      </div>
+      </div>
+      <Divider/>
```
Comment (AniruddhaGawali) [inline frontend/src/pages/index.js:25]: Give component proper name WaveDivider

File: frontend/src/pages/index.js [hunk 5]
```diff
@@ -10,9 +12,19 @@ export default function Home() {
...
       </Head>
-      <main className={styles.main}>
-        <h1 className="text-red-500 text-5xl">Hello World</h1>
-      </main>
+      <div className="bg-black">
+      <Divider/>
+      <div>
+      <div className="text-4xl left-24 m-10 truncate align-middle text-gray-300 mb-5"> RECENT BLOGS </div>
+      <div className="flex justify-evenly items-center mb-10 mt-10">
+          <Card />
```
Comment (AniruddhaGawali) [inline frontend/src/pages/index.js:20]: Give component proper name Recent Blog Card

**Review by AniruddhaGawali — CHANGES_REQUESTED** [review]
Read the added comment and do the changes

File: frontend/src/pages/index.js (hunk 4 above)
Comment (Mohitraut07) [reply frontend/src/pages/index.js:25]: ok

File: frontend/src/pages/index.js (hunk 5 above)
Comment (Mohitraut07) [reply frontend/src/pages/index.js:20]: sure

**Pull request closed without merging**